app/controllers/users_controller.rb

```diff
@@ -1,7 +1,9 @@
 class UsersController < ApplicationController
   def index
-    @user = User.last_posts(current_user.name)
+    @user = User.all
   end
 
-  def show; end
+  def show
+    @user = User.find_by_id(params[:id])
+  end
 end
```

Comment:
- [ ] Remember you need to have a recent_post method that will display only 3recent posts of the user
and when a person wants to view all the user posts will hit `show all posts` link